pkg/runtime/local/local.go

```diff
@@ -80,6 +81,10 @@ func (r *Runtime) ParamDescs() params.ParamDescs {
 	return nil
 }
 
+func (r *Runtime) GetGadgetInfo(desc gadgets.GadgetDesc, pars *params.Params, args []string) (*runTypes.GadgetInfo, error) {
+	return desc.(runTypes.RunGadgetDesc).GetGadgetInfo(pars, args)
```

Comment:
Would that crash is the grpc client is calling this method on a different gadget?
```suggestion
	runDesc, ok := desc.(runTypes.RunGadgetDesc)
	if !ok {
		return nil, fmt.Errorf("GetGadgetInfo not supported for gadget %s", desc.Name())
	}
	return runDesc.GetGadgetInfo(pars, args)
```

Reply:
I don't think this is a big issue as the client should never call this. Anyway, I added the check to be sure it doesn't cause issues.